[2025-05-31T23:56:59.481+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: entrenamiento_modelos_regresion.entrenar_y_promover manual__2025-05-31T23:56:57.953473+00:00 [queued]>
[2025-05-31T23:56:59.494+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: entrenamiento_modelos_regresion.entrenar_y_promover manual__2025-05-31T23:56:57.953473+00:00 [queued]>
[2025-05-31T23:56:59.495+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-31T23:56:59.511+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): entrenar_y_promover> on 2025-05-31 23:56:57.953473+00:00
[2025-05-31T23:56:59.517+0000] {standard_task_runner.py:60} INFO - Started process 246 to run task
[2025-05-31T23:56:59.523+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'entrenamiento_modelos_regresion', 'entrenar_y_promover', 'manual__2025-05-31T23:56:57.953473+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/entrenamiento_modelos_regresion_dag.py', '--cfg-path', '/tmp/tmp6nfm04vv']
[2025-05-31T23:56:59.527+0000] {standard_task_runner.py:88} INFO - Job 3: Subtask entrenar_y_promover
[2025-05-31T23:56:59.607+0000] {task_command.py:423} INFO - Running <TaskInstance: entrenamiento_modelos_regresion.entrenar_y_promover manual__2025-05-31T23:56:57.953473+00:00 [running]> on host 76da0f717085
[2025-05-31T23:56:59.738+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='fede' AIRFLOW_CTX_DAG_ID='entrenamiento_modelos_regresion' AIRFLOW_CTX_TASK_ID='entrenar_y_promover' AIRFLOW_CTX_EXECUTION_DATE='2025-05-31T23:56:57.953473+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-31T23:56:57.953473+00:00'
[2025-05-31T23:57:03.980+0000] {logging_mixin.py:188} WARNING - /opt/***/dags/entrenamiento_modelos_regresion_dag.py:56 SettingWithCopyWarning:
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2025-05-31T23:57:04.002+0000] {logging_mixin.py:188} WARNING - /opt/***/dags/entrenamiento_modelos_regresion_dag.py:59 SettingWithCopyWarning:
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2025-05-31T23:57:04.004+0000] {logging_mixin.py:188} WARNING - /opt/***/dags/entrenamiento_modelos_regresion_dag.py:61 SettingWithCopyWarning:
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2025-05-31T23:57:04.006+0000] {logging_mixin.py:188} WARNING - /opt/***/dags/entrenamiento_modelos_regresion_dag.py:63 SettingWithCopyWarning:
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2025-05-31T23:57:04.048+0000] {logging_mixin.py:188} WARNING - 2025/05/31 23:57:04 INFO mlflow.tracking.fluent: Experiment with name 'entrenamiento_orquestado' does not exist. Creating a new experiment.
[2025-05-31T23:57:04.201+0000] {logging_mixin.py:188} WARNING - 2025/05/31 23:57:04 WARNING mlflow.utils.git_utils: Failed to import Git (the Git executable is probably not on your PATH), so Git SHA is not available. Error: Failed to initialize: Bad git executable.
The git executable must be specified in one of the following ways:
    - be included in your $PATH
    - be set via $GIT_PYTHON_GIT_EXECUTABLE
    - explicitly set via git.refresh(<full-path-to-git-executable>)

All git commands will error until this is rectified.

This initial message can be silenced or aggravated in the future by setting the
$GIT_PYTHON_REFRESH environment variable. Use one of the following values:
    - quiet|q|silence|s|silent|none|n|0: for no message or exception
    - warn|w|warning|log|l|1: for a warning message (logging level CRITICAL, displayed by default)
    - error|e|exception|raise|r|2: for a raised exception

Example:
    export GIT_PYTHON_REFRESH=quiet
[2025-05-31T23:57:08.809+0000] {logging_mixin.py:188} WARNING - 2025/05/31 23:57:08 WARNING mlflow.models.model: Model logged without a signature and input example. Please set `input_example` parameter when logging the model to auto infer the model signature.
[2025-05-31T23:57:08.962+0000] {logging_mixin.py:188} WARNING - 2025/05/31 23:57:08 INFO mlflow.tracking._tracking_service.client: üèÉ View run RandomForest at: http://mlflow:5000/#/experiments/1/runs/1f97d0eb28d7430a99176cf531bdd41f.
[2025-05-31T23:57:08.962+0000] {logging_mixin.py:188} WARNING - 2025/05/31 23:57:08 INFO mlflow.tracking._tracking_service.client: üß™ View experiment at: http://mlflow:5000/#/experiments/1.
[2025-05-31T23:57:12.841+0000] {logging_mixin.py:188} WARNING - 2025/05/31 23:57:12 WARNING mlflow.models.model: Model logged without a signature and input example. Please set `input_example` parameter when logging the model to auto infer the model signature.
[2025-05-31T23:57:12.917+0000] {logging_mixin.py:188} WARNING - 2025/05/31 23:57:12 INFO mlflow.tracking._tracking_service.client: üèÉ View run CatBoost at: http://mlflow:5000/#/experiments/1/runs/8c04eb98daab4cfcb49a4af61bd1f1b4.
[2025-05-31T23:57:12.917+0000] {logging_mixin.py:188} WARNING - 2025/05/31 23:57:12 INFO mlflow.tracking._tracking_service.client: üß™ View experiment at: http://mlflow:5000/#/experiments/1.
[2025-05-31T23:57:15.591+0000] {logging_mixin.py:188} WARNING - 2025/05/31 23:57:15 WARNING mlflow.models.model: Model logged without a signature and input example. Please set `input_example` parameter when logging the model to auto infer the model signature.
[2025-05-31T23:57:15.825+0000] {logging_mixin.py:188} WARNING - 2025/05/31 23:57:15 INFO mlflow.tracking._tracking_service.client: üèÉ View run SVR at: http://mlflow:5000/#/experiments/1/runs/8c3fe93033694453be5a8eb2482ae9e9.
[2025-05-31T23:57:15.826+0000] {logging_mixin.py:188} WARNING - 2025/05/31 23:57:15 INFO mlflow.tracking._tracking_service.client: üß™ View experiment at: http://mlflow:5000/#/experiments/1.
[2025-05-31T23:57:15.922+0000] {logging_mixin.py:188} WARNING - Successfully registered model 'mejor_modelo_orquestado'.
[2025-05-31T23:57:15.976+0000] {logging_mixin.py:188} WARNING - 2025/05/31 23:57:15 INFO mlflow.store.model_registry.abstract_store: Waiting up to 300 seconds for model version to finish creation. Model name: mejor_modelo_orquestado, version 1
[2025-05-31T23:57:15.977+0000] {logging_mixin.py:188} WARNING - Created version '1' of model 'mejor_modelo_orquestado'.
[2025-05-31T23:57:15.978+0000] {logging_mixin.py:188} WARNING - /opt/***/dags/entrenamiento_modelos_regresion_dag.py:99 FutureWarning: ``mlflow.tracking.client.MlflowClient.get_latest_versions`` is deprecated since 2.9.0. Model registry stages will be removed in a future major release. To learn more about the deprecation of model registry stages, see our migration guide here: https://mlflow.org/docs/latest/model-registry.html#migrating-from-stages
[2025-05-31T23:57:15.991+0000] {logging_mixin.py:188} WARNING - /opt/***/dags/entrenamiento_modelos_regresion_dag.py:102 FutureWarning: ``mlflow.tracking.client.MlflowClient.transition_model_version_stage`` is deprecated since 2.9.0. Model registry stages will be removed in a future major release. To learn more about the deprecation of model registry stages, see our migration guide here: https://mlflow.org/docs/latest/model-registry.html#migrating-from-stages
[2025-05-31T23:57:16.024+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-31T23:57:16.042+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=entrenamiento_modelos_regresion, task_id=entrenar_y_promover, execution_date=20250531T235657, start_date=20250531T235659, end_date=20250531T235716
[2025-05-31T23:57:16.114+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-31T23:57:16.137+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
